code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1046,6 +1046,51 @@
 	..()
 	. = 1
 
+/datum/reagent/medicine/clotagen
+	name = "Clotagen"
+	description = "Encourages coagulation to stop wounds from bleeding."
+	color = "#FA7DB5"
+	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
+	taste_description = "a roll of gauze"
+	metabolization_rate = 0.3 * REAGENTS_METABOLISM
+	overdose_threshold = 20
+
+/datum/reagent/medicine/clotagen/on_mob_life(mob/living/carbon/M)
```

Comment:
Drinking or injecting something that supposedly turns blood into tar sounds really stupid. 

I think this reagent should only work through contact method, with all other methods either have no effect or a negative effect.

Reply:
Would be interesting if it had some special interaction with oxygen or nitrogen to make it like "it hardens on contact with the outside air" or something

Reply:
> Drinking or injecting something that supposedly turns blood into tar sounds really stupid. A doctor would not try to introduce that to the whole circulatory system, they would try to keep it very very local.
> 
> I think this reagent should only work through contact method(patches), with all other methods either have no effect or a negative effect.

I'm not sure about this, I get your point but I don't want it to be limited to 1 type of application, then it'd probably be mostly unused.